integration-tests/contracts/ccipreader_test.go

```diff
@@ -2,6 +2,7 @@ package contracts
 
 import (
```

Comment:
This file is in odd directory, isn't it? Other "component" level integration tests are here https://github.com/smartcontractkit/chainlink/tree/develop/core/capabilities/ccip/ccip_integration_tests

Reply:
This was moved in this PR: https://github.com/smartcontractkit/chainlink/pull/15357 

maybe @asoliman92 can shed some light?